src/BuildToolsSmokeTests/Checks.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Google.Protobuf.WellKnownTypes;
+using System;
+using Xunit;
+
+namespace ProtoBuf
+{
+    public class Usage
+    {
+        [Theory]
+        [InlineData(typeof(Vanilla.Foo), nameof(Vanilla.Foo.I), typeof(double?))]
+        [InlineData(typeof(Enabled.Foo), nameof(Enabled.Foo.I), typeof(double?))]
+        [InlineData(typeof(Disabled.Foo), nameof(Disabled.Foo.I), typeof(DoubleValue))]
+        public void CheckPropertyType(Type source, string name, Type valueType)
```

Comment:
can you add an additional test here, that checks (at runtime) whether the property is marked with the expected attribute? (and check you can get it working locally; you might need to change the Version in the csproj)

Reply:
Added tests on all `wrappers.proto` types, but for some unknown to me reason build of project fails:
```
1>C:\Code\forked\protobuf-net\src\BuildToolsSmokeTests\Checks.cs(48,63,48,74): error CS0117: 'Foo' does not contain a definition for 'stringItems'      
```
here is a line where it fails
![image](https://user-images.githubusercontent.com/31598696/218263448-e1105827-6f2b-486f-b99c-07dee4acb62f.png)

and I see property exists
![image](https://user-images.githubusercontent.com/31598696/218263457-b7c9f2c3-4d01-4dc5-9670-9b57bcee4292.png)

Can you, please, take a look - what am I doing wrong?

Reply:
Are you using the console and `dotnet test`? The IDE hates this setup

Reply:
same thing. I did not do any change to analyzer, but rebuilt using your readme and still getting this error
![image](https://user-images.githubusercontent.com/31598696/218264348-dd19805d-abdd-4ab5-a585-7895f2264969.png)

Reply:
silly me - for some reason .proto field was not changed.
It is fixed and works!